src/delay-queue.ts

```diff
@@ -54,6 +57,11 @@ export class DelayQueue<T = unknown> extends RxQueue<T> {
     super.unsubscribe()
   }
 
+  public clear () {
```

Comment:
How about to use `reset()` instead of `clear()`?

I just feel that the `reset()` would be better, but you can make the final decision.